src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/AuthorizationFunctionalTest.java

```diff
@@ -13,41 +14,21 @@
 import uk.gov.hmcts.reform.judicialapi.idam.IdamOpenIdClient;
 import uk.gov.hmcts.reform.lib.client.response.S2sClient;
 
-import javax.annotation.PostConstruct;
-
-import static org.apache.commons.lang.RandomStringUtils.randomAlphanumeric;
-
 @ContextConfiguration(classes = {TestConfigProperties.class, Oauth2.class})
 @ComponentScan("uk.gov.hmcts.reform.judicialapi")
 @TestPropertySource("classpath:application-functional.yaml")
 @Slf4j
 public class AuthorizationFunctionalTest {
 
-    @Value("${s2s-url}")
-    protected String s2sUrl;
-
-    @Value("${s2s-name}")
-    protected String s2sName;
-
-    @Value("${s2s-secret}")
-    protected String s2sSecret;
-
     @Value("${targetInstance}")
     protected String jrdApiUrl;
 
     protected static JudicialApiClient judicialApiClient;
 
     protected static IdamOpenIdClient idamOpenIdClient;
 
-    @Autowired
-    protected TestConfigProperties configProperties;
-
     protected static String s2sToken;
 
-    public static final String EMAIL = "EMAIL";
-    public static final String CREDS = "CREDS";
-    public static final String EMAIL_TEMPLATE = "test-user-%s@jrdfunctestuser.com";
-    public static final String ROLE_JRD_ADMIN = "jrd-admin";
     public static final String ROLE_JRD_SYSTEM_USER = "jrd-system-user";
```

Comment:
This is unused can you remove it?

Reply:
It actually is used in `JudicialUsersFunctionalTest` class...